main.js

```diff
@@ -70,4 +59,24 @@ $(document).ready(function(){
  });
 
 
+function element(name ,price){
+  return '<div class="col-md-3">' +
+  '<div class="book-box">' +
+  '<img src= "image/book.jpg" class="img-j1" alt="book">' +
+  '<span class="p-name-price">' + 'Name :' + name + '</span>' +
+  '<span class="p-name-price">' + 'Price :' + price + '</span>' +
+  '<button type="button" id="add-cart" class="add-cart-button btn btn-primary  btn-block">Add to Cart</button>' +
+  '</div>' + '</div>' ;}
 
+
+function check(){
+  localStorage.setItem('islogin', false);
+  if(localStorage.getItem('islogin')==false){
```

Comment:
- based on your implementation its always false.
- and getItems return string, and u are comparing string with boolean.
- u need to use triple equal for deep comparing.